problems/variables.js

```diff
@@ -7,6 +7,12 @@
 //  * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
 //  * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
 
+let birthYear = '1977'
+let futureYear = '2021'
+let minimumAge = '44' 
```

Comment:
Instead of telling me what min and max age are, we want to see an equation.

If you think about it, the minimum age changes when we change birthYear or futureYear so we want to have an equation so our minimumAge and maximumAge are always correct.